docs/src/user/overview.rst

```diff
@@ -0,0 +1,26 @@
+********
+Overview
+********
+
+.. The goal of this document is to provide an overview of the content present in the user guide.
+.. It is not meant to replace the general getting started or duplicate the content in the user guide
+.. sections.
+
+In this guide, we explain the concepts of Oríon in depth. A light weight introduction is available
+in :doc:`/install/gettingstarted` to familiarize yourself with the project and its concepts.
+
+The user guide is organized in the following sections:
+
+* :doc:`/user/script`. Describes how you can integrate Oríon to your existing scripts and
```

Comment:
Maybe drop all the describes and start with 'How [...]'.
I wounder if using `:attr: desc.` like in [config documentation](https://raw.githubusercontent.com/Epistimio/orion/develop/docs/src/user/config.rst) would give a better layout.

Reply:
I tried to remove the *Describes* but it's a bit awkward without them.
I also tried the attributes but it takes way too much space for a simple index. It would have been fine if the bold meta-title could be a link but unfortunately, restructuredText doesn't allow it :(.